src/gui/addnewtorrentdialog.cpp

```diff
@@ -37,6 +37,7 @@
 #include <QUrl>
 #include <QVector>
 
+#include "autoexpandabledialog.h"
```

Comment:
https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#7-header-inclusion-order

Reply:
sorry but I really can't see the problem, they are sorted alphabetically, Can you please explain the problem.

Reply:
qBittorrent headers should be started from *base* headers as it was before your changes.

Reply:
ok got it, base headers will come first.